nncf/experimental/common/pruning/operations.py

```diff
@@ -93,16 +92,13 @@ def mask_propagation(cls, node: NNCFNode, graph: NNCFGraph,
                          tensor_processor: Type[NNCFPruningBaseTensorProcessor]) -> None:
         node.data['output_mask'] = None
 
-
 class OutputPruningOp(BasePruningOp):
-    @classmethod
-    def accept_pruned_input(cls, node: NNCFNode) -> bool:
-        return True
-
```

Comment:
I think this method is required for proper mask propagation

Reply:
It's not used anywhere. propose to add when will actually needed

Reply:
Ah, this method is for legacy node selector algo, you are right